.github/workflows/ci.yml

```diff
@@ -93,9 +93,9 @@ jobs:
       run: ./scripts/patch_dependencies.sh
     - name: Run tests
```

Comment:
the "Run tests" can be renamed to something more apt to reflect the proposed change.